generator-jhipster-go/generators/server/templates/go/go/customlogger/customlogger.go

```diff
@@ -36,17 +36,17 @@ func createLogger(fname string) *loggers {
 }
 
 func Printfun(level string,msg string) {
-    logger := GetInstance()
-    if(level=="info"){
-    logger.InfoLogger.Println(msg)
-    }
-
-    if(level=="warn"){
-     logger.WarningLogger.Println(msg)
-    }
-
-    if(level=="error"){
-    logger.ErrorLogger.Println(msg)
-    }
+    // logger := GetInstance()
+    // if(level=="info"){
+    // logger.InfoLogger.Println(msg)
+    // }
+
+    // if(level=="warn"){
+    //  logger.WarningLogger.Println(msg)
+    // }
+
+    // if(level=="error"){
+    // logger.ErrorLogger.Println(msg)
+    // }
```

Comment:
why is this commented? we must log into file as well.

Reply:
we are reading container logs i believe, so commented them so of now!
will check and revert back!